src/fragments/Projects/Settings/Sections/Billing/BillingPlan/BillingPlan.tsx

```diff
@@ -283,3 +250,60 @@ const getPlanData = (plan: BillingPlanType, endPlanDate?: string): PlanData => {
     },
   }[plan];
 };
+
+export const Banners = ({ isLoading }: LoadingProps) => {
+  const { subscription, billingPlanType, paymentMethod } = useBillingContext();
+  const shouldShowCancellationBanner = useMemo(() => {
+    if (subscription.data?.endDate) {
+      const targetTime = DateTime.fromISO(subscription.data.endDate);
+      const currentTime = DateTime.now();
+      const diff = targetTime.diff(currentTime);
+
+      return Math.floor(diff.as('months')) <= 1;
+    }
+
+    return null;
+  }, [subscription.data?.endDate]);
+
+  const trialEndDate = useMemo(() => {
+    if (subscription.data?.trialEndDate) {
+      return dateFormat({
+        dateISO: subscription.data.trialEndDate,
+        format: DateTime.DATE_FULL,
+      });
+    }
+
+    return '';
+  }, [subscription.data?.trialEndDate]);
+
+  const endPlanDate = useMemo(() => {
+    if (subscription.data?.endDate) {
+      return dateFormat({
+        dateISO: subscription.data.endDate,
+        format: DateTime.DATE_FULL,
+      });
+    }
+
+    return '';
+  }, [subscription.data?.endDate]);
+
+  return (
+    <>
+      {shouldShowCancellationBanner && !isLoading && (
+        <AlertBox size="sm" className="font-medium">
+          Your Pro Plan is expiring. You will be converted to a Free plan on{' '}
+          {endPlanDate}.
```

Comment:
Maybe something like:
{`Your Pro Plan is expiring. You will be converted to a Free plan on ${endPlanDate}.`}